doc/EMAIL.md

```diff
@@ -1,24 +1,22 @@
+[This](https://github.com/publiclab/plots2/blob/master/script/mailman_server) file contains the script for starting the mailman server. It is using POP3 (Post Office Protocol version 3) to receive emails from a remote server to a local email client. All the configurations regarding this are to be done in [mailman script](https://github.com/publiclab/plots2/blob/master/script/mailman_server). 
 
-After configurations are done we are good to start the `mailman` server. For starting the server run the `mailman_server` script file present inside script folder.
+After configurations are done we are good to start the `mailman` server. For starting the server run the `mailman_server` script file present inside the script folder.
 
 > ruby script/mailman_server
 
-Above command will start the server and then the mailman server will do polling in every 60 seconds(which is by default) to check for the incoming mail. However polling time can be changed by setting value to `Mailman.config.poll_interval`(more details can be found at its official [mailman user guide](https://github.com/mailman/mailman/blob/master/USER_GUIDE.md)). Logs for mailman can be seen oin Mailman log file which is present in `log` folder.
-    3.
+Above command will start the server and then the mailman server will do polling every 60 seconds(which is by default) to check for the incoming mail. However polling time can be changed by setting the value to `Mailman.config.poll_interval`(more details can be found at its' official [mailman user guide](https://github.com/mailman/mailman/blob/master/USER_GUIDE.md)). Logs for mailman can be seen in Mailman log file which is present in the `log` folder.
 
- 
 ## Email Settings
 
 Users can modify **Email Settings** by visiting [https://publiclab.org/settings](https://publiclab.org/settings). Settings for each user are saved by using UserTags on their profiles. 
 For each setting, there is an associated tag with it. Below is the table showing associated tags with setting. 
 
-We consider absence of tag as true and presence of tag as false for setting, so whenever a user turn-off a setting, corressponding user-tag is generated and turning setting on again will delete the tag.  
-So, while notifying a user UserTag is checked in different files and user is notified depending on the presence or absence of the tag.
+We consider absence of **tag** as true and presence of **tag** as false for setting, so whenever a user turns **off** a setting, a corresponding user-tag is generated and turning setting **on** again will delete the tag.  
+So, while notifying a user **UserTag** is checked in different files and _user_ is notified depending on the presence or absence of the tag.
```

Comment:
The italics on `_user_` probably aren't needed here. You could change this to say "the user is notified" so it'd be easier to understand.